lib/printer_util.go

```diff
@@ -1,28 +1,16 @@
 package lib
```

Comment:
I don't see a problem on this specific case.

Reply:
Before, it made a little more sense to have the vars with the init func in that file, but since the `init` func got deleted dedicating an entire file to 4 vars is not something you see in a clean go code base.

Reply:
I am convinced that since only `runner.go` and `runner_test.go` use these variables, they can live in `runner.go`. 

Though, the number of variables, or constants, or functions, does not determine the cleanliness of the code base, but the separation of concerns of the files yes.